include/exadg/time_integration/time_int_multistep_base.h

```diff
@@ -65,12 +63,6 @@ class TimeIntBDFBase : public TimeIntBase
   void
   timeloop_steady_problem();
 
-  /*
-   * Setters and getters.
-   */
-  double
-  get_scaling_factor_time_derivative_term() const;
-
```

Comment:
For clarity: Is there a reason why you removed this function from the interface, while you made the function `update_time_integrator_constants()` below pure virtual in the present PR?

This seems to be the main change of this PR (and could also be briefly mentioned in the PR description).

Reply:
```update_time_integrator_constants()``` is needed for all multistep methods. The scaling factor for the time derivative term is only needed for BDF (at least, I didn't use it in the Adams methods). I would suggest proceeding like this and if you feel it would be helpful also for Adams methods move it back to the base class. Would this be OK?

Reply:
A question is also whether `update_time_integrator_constants()` should still be in the base class, given that we no longer have the member variables for the time integrator constants in the base class? As you suggest for the other question above, we can also keep this question in mind for the upcoming changes and then decide for a "final" design.